lua/weapons/grav_blaster.lua

```diff
@@ -0,0 +1,232 @@
+if SERVER then
+	resource.AddSingleFile("materials/entities/grav_blaster.png")
+	AddCSLuaFile()
+end
+
+SWEP.PrintName = "Grav Blaster"
+SWEP.Category = TASWeapons.Category
+SWEP.Author = "jason"
```

Comment:
Who is this?

Reply:
must be a hacker

Reply:
It's me

Reply:
@yogwoggf use your steam or github username please